examples/yt_camera/main.cpp

```diff
@@ -275,7 +281,21 @@ int main(int argc, char** argv)
 
                 LOG(DEBUG) << "Posting object stream";
 
-                status = client.uploadObjectStream(accountId, instrumentId, os, BLOB_TMP_FILE);
+                Mat blob = Mat(frame, r);
+
+#if BLOB_FROM_FILE
+                imwrite(BLOB_TMP_FILE, blob, compression_params);
+                status = client.uploadObjectStream(accountId, instrumentId,
+                                                   os, prc::Payload(BLOB_TMP_FILE));
+#else
+                std::vector<uchar> buf;
+                bool rv = imencode(".jpg", blob, buf, compression_params);
```

Comment:
HW supported, if available would be the best.
For now, consider this example as example for PC, because of embedded we'd like to minimize number of third-party libraries built to ease SDK use.